corehq/ex-submodules/casexml/apps/case/tests/test_property_changed_date.py

```diff
@@ -1,12 +1,18 @@
 from __future__ import absolute_import
+
 from datetime import datetime
-import pytz
 
+import pytz
```

Comment:
nit: you changed the imports, but didn't make them alphabetical

Reply:
I used https://github.com/timothycrosley/isort, but I guess it didn't do the right thing...

Reply:
weird. Maybe it puts imports before from .. import ?